docs/dm/zh_CN/restrictions.md

```diff
@@ -0,0 +1,20 @@
+使用限制
+===
+
+### 数据库版本
+
+- 5.5 < MySQL version < 5.8
+- MariaDB version >= 10.1.2
+
+### DDL 语法
+
+DM 使用 TiDB parser 解析处理 DDL statement，所以仅支持 TiDB parser 支持的 DDL 语法, 详情请参阅 [TiDB DDL 语法支持](https://github.com/pingcap/docs-cn/blob/master/sql/ddl.md)
+
+### 分库分表
+
+参阅 [分库分表数据同步](./shard-table/data-synchronization.md)
```

Comment:
past limits of shard table here？  and create a file for  limits of shard table

Reply:
I split `shard-table/data-synchronization.md` to `shard-table/restrictions.md` and `shard-table/handle-DDL-lock.md`.

I prefer add links rather than paste because I do not want to maintain multi parts in docs